.vscode/settings.json

```diff
@@ -1,17 +1,29 @@
 {
-    "editor.tabSize": 2,
-    "editor.insertSpaces": true,
-    "editor.detectIndentation": false,
-    "typescript.tsdk": "node_modules/typescript/lib",
-    "docker.commands.runInteractive": [
-      {
-        "label": "Default run command",
-        "template": "docker run --rm -it ${exposedPorts} ${tag}"
-      },
-      {
-        "label": "Run docs container",
-        "template": "docker run --rm -it ${exposedPorts} -v ${workspaceFolder}/docs:/docs ${tag}",
-        "match": "squidfunk"
-      }
-    ]
+  "editor.tabSize": 2,
+  "editor.renderFinalNewline": false,
+  "files.insertFinalNewline": false,
+  "editor.insertSpaces": true,
+  "editor.detectIndentation": false,
+  "editor.guides.bracketPairs": true,
```

Comment:
Isn't this related to UI rather than formatting? If that's indeed the case, let's remove it as different folks might have different preferences that don't affect our code base.